### PR TITLE
Addnewplaydate

### PR DESCRIPTION
**Pull request by ctunakan**

### CODE REVIEW

File: src/views/Calendar.jsx [hunk 1]
```diff
@@ -51,7 +56,7 @@ const Calendar = () => {
       })
       .then(() => axios.get(`http://localhost:3001/api/getpacks?userId=${7}`))
       .then((packData) => {
-        // console.log(packData.data);
+        console.log(packData.data);
```
Comment (neb-abera) [inline src/views/Calendar.jsx]: remove console.log

Comment (ctunakan) [reply src/views/Calendar.jsx]: I removed it thank you!

**Review by neb-abera — APPROVED** [review]

File: src/components/Calendar/AddPlaydate.jsx [hunk 2]
```diff
@@ -1,48 +1,78 @@
...
+    return axios
+      .post('http://localhost:3001/api/addplaydate', {
+        packId: packChoiceId,
+        userId: 7, // change later to by whatever user is logged in
+        playdateBody: playdateInfo,
+        startTime: playStartTime,
+        endTime: playEndTime
+      })
+      .then(() => {
+        console.log('playdate added!');
```
Comment (neb-abera) [inline src/components/Calendar/AddPlaydate.jsx:37]: remove console log

File: src/components/Calendar/AddPlaydate.jsx [hunk 3]
```diff
@@ -1,48 +1,78 @@
...
+        closeAddModal();
+      });
+    console.log('playdate added');
```
Comment (neb-abera) [inline src/components/Calendar/AddPlaydate.jsx:40]: remove console log

**Pull request merged**